core/controllers/acl_decorators_test.py

```diff
@@ -4737,6 +4737,38 @@ def test_can_edit_skill(self):
             self.assertEqual(response['entity_type'], 'skill')
         self.logout()
 
+    def test_can_submit_images_to_questions(self):
+        self.login(self.CURRICULUM_ADMIN_EMAIL)
+        skill_id = skill_services.get_new_skill_id()
+        self.save_new_skill(skill_id, self.admin_id, description='Description')
+        with self.swap(self, 'testapp', self.mock_testapp):
+            response = self.get_json('/mock_edit_entity/%s/%s' % (
+                feconf.IMAGE_CONTEXT_QUESTION_SUGGESTIONS, skill_id))
+            self.assertEqual(response['entity_id'], skill_id)
+            self.assertEqual(response['entity_type'], 'question_suggestions')
+        self.logout()
+
+    def test_cannot_submit_images_to_questions_by_unaunthenticated_users(self):
```

Comment:
"unauthenticated" is misspelled.

Maybe clearer to say "test_unauthenticated_users_cannot_submit_images_to_questions"